pkg/machine/qemu/machine.go

```diff
@@ -248,7 +247,26 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	if err := v.startHostNetworking(); err != nil {
 		return errors.Errorf("unable to start host networking: %q", err)
 	}
+
+	rtPath, err := getRuntimeDir()
+	if err != nil {
+		return err
+	}
+	if !rootless.IsRootless() {
+		rtPath = "/run"
+	}
```

Comment:
Should this be done inside getRuntimeDir()? This could be easily be missed when getRuntimeDir is used in another place and you end up with different paths.

Reply:
Actually on second look, it already is ... this code must have been old.  Nice catch